lib/task-data/tasks/install-coreos.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
         comport: 'ttyS0',
         hostname: 'coreos-node',
         installDisk: '/dev/sda',
-        completionUri: 'pxe-cloud-config.yml'
+        completionUri: 'pxe-cloud-config.yml',
+        repo: '{{api.server}}/coreos'
```

Comment:
sure, it works, just modify this repo address to the external one while trigger it. This default repo value is just want to gain back compatibility.